server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -459,7 +460,8 @@ public static String jobConfig(String jobKey, String cfgName) {
             // Async job defaults and scheduling
             this.put(ASYNC_JOBS_THREADS, "10");
 
-            this.put(jobConfig(ManifestCleanerJob.JOB_KEY, ASYNC_JOBS_JOB_ENABLED), "true");
+            this.put(jobConfig(ExpiredPoolsCleanupJob.JOB_KEY, ASYNC_JOBS_JOB_SCHEDULE),
```

Comment:
One thing I forgot: you didn't remove the `ExpiredPoolsJob.class.getName(),` entry from the `DEFAULT_TASK_LIST` in this file.